### PR TITLE
Handle negative phase0 in LCModel .COORD files

### PR DESCRIPTION
**Pull request by xaker00**
Negative three-digit phase0 values would crash the parser. Now fixed with regex.

Updated tests and sorted imports using ruff in modified files.

### CODE REVIEW

File: tests/test_data/lcmodel/liver.COORD [hunk 1]
```diff
@@ -19,7 +19,7 @@
    5 lines in following misc. output table
   FWHM = 0.297 ppm    S/N =  24                                            
   Data shift =-0.019 ppm                                                   
-  Ph:  36 deg       0.3 deg/ppm                                            
+  Ph:-136 deg       0.3 deg/ppm
```
Comment (xaker00) [inline tests/test_data/lcmodel/liver.COORD:22]: This .COORD file is only used in one test. I modified it to include the edge case that was crashing previously.
